index.js

```diff
@@ -0,0 +1,35 @@
+const twemoji = require('twemoji');
+const emoji = require('node-emoji');
```

Comment:
I didn't realize that you need to convert to unicode emoji first and then pass that to twemoji... maybe it would be better to call this package `marked-twemoji` then?

Reply:
Within the first 72 hours, you can unpublish a package (see [unpublish policy](https://docs.npmjs.com/policies/unpublish)), via command

```
npm unpublish marked-emoji
```

and then you can republish it with a different name.

Just suggesting this because someone (maybe me) might also want to create a marked-emoji extension that only converts the `:<...>:` syntax into a unicode emoji (just with `node-emoji`).

Reply:
Great idea, thanks!

Reply:
Note that I made a mistake when publish `marked-twemoji@1.0.0-SNAPSHOT`, I had to unpuslish it
I will re-publish this version tomorrow (npm blocked publish for 24hours)

Reply:
ok or you can also just change `-SNAPSHOT` to something else, e.g. `-SNAPSHOT2` ;)

Reply:
I can do that if you want to test it tomorrow yes!

Reply:
no need from my side 🤓